src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -423,6 +427,11 @@ const initialise = (
 				options: {
 					ajaxUrl: window.guardian.config.page.ajaxUrl ?? '',
 					pv: window.guardian.ophan.pageViewId,
+					enableV2Endpoint: isUserInVariant(
+						newHeaderBiddingEndpoint,
+						'variant',
+					),
+					ajaxUrlV2: `//performance-events.code.dev-guardianapis.com/header-bidding`,
```

Comment:
Could you expand on this endpoint please? How did you form it in this way?

Reply:
`performance-events.code.dev-guardianapis.com` (and the prod version `performance-events.guardianapis.com`) is setup in fastly already for commercial-metrics and core-web-vitals, I've added the `header-bidding` path within fastly to send data to the datalake. Perhaps I can go over it in a dev time?

Reply:
Oh ok! Yes that would be great please 🙂